File: Classes/17th Nov/prime/prime_v2.c
//import java.util.Scanner;
#include<stdio.h>
#include<stdlib.h>
#include<stdbool.h>
#include<math.h>
#include<time.h>
// stdin  stdout stderr


    int main( char * parameters[] )
	{

		int num;
		scanf("%d",&num);
		bool factor_found_flag;
		factor_found_flag = false;
		clock_t start_time = clock();
		int cff;
		for ( cff = 2; cff < sqrt(num); cff++)
		{
		    if ( num % cff == 0 )
			{
			     factor_found_flag = true;
			     break;
			}
		}
		// here
		clock_t stop_time = clock();
		double duration = ( stop_time - start_time) / 1000.0;
		printf("%s , time taken = %.3f",
		factor_found_flag == true ? "Composite" : "Prime", duration);

		return EXIT_SUCCESS;

}

File: Classes/17th Nov/prime/prime_v2.java
import java.util.Scanner;
        
class prime_v2
{
	public static void main( String [] parameters )
	{
		Scanner kbrd = new Scanner ( System.in );
		int num;
		num = kbrd.nextInt();
		boolean factor_found_flag;
		factor_found_flag = false;
		long start_time = System.currentTimeMillis();
		int cff;
		for ( cff = 2; cff < Math.sqrt(num); cff++)
		{
		    if ( num % cff == 0 )
			{
			     factor_found_flag = true;
			     break;
			}
		}
		// here
		long stop_time = System.currentTimeMillis();
		double duration = ( stop_time - start_time) / 1000.0;
		System.out.printf("%s , time taken = %.3f", 
		factor_found_flag == true ? "Composite" : "Prime", duration);
	}
}